MANIFEST.in

```diff
@@ -40,6 +37,9 @@ global-exclude .DS_Store
 global-exclude .git*
 global-exclude \#*
 
+# GH 39321
```

Comment:
Could you include a brief explainer as well, instead of only the issue reference?